dev-docs/publisher-api-reference.md

```diff
@@ -1231,6 +1233,16 @@ Prebid core adds a timeout on XMLHttpRequest request to terminate the request on
 pbjs.setConfig({ disableAjaxTimeout: true });
 {% endhighlight %}
 
+#### Set Timeout Buffer
+
+<a name="setConfig-timeoutBuffer" />
+
+Prebid core adds a timeout buffer to extend the time that bidders have to return a bid after the auction closes. It's used to offset the "time slippage" of the setTimeout behavior in browsers. Prebid.js defaults this value to 400ms. However, you have the option to update using below snippet.
```

Comment:
Suggested rewording: 
Prebid core adds a timeout buffer to extend the time that bidders have to return a bid after the auction closes. This buffer is used to offset the "time slippage" of the setTimeout behavior in browsers. Prebid.js sets the default value to 400ms. You can change this value by setting `timeoutBuffer` to the amount of time you want to use. The following example sets the buffer to 300ms.